src/server.ts

```diff
@@ -11,6 +12,26 @@ app.get("/", (_, res) => {
   res.redirect("/admin");
 });
 
+app.use((req, res, next) => {
+  const cookieHeader = req.headers.cookie || '';
+  const tokenMatch = cookieHeader.match(/payload-token=([^;]+)/);
+
+  if (tokenMatch) {
+    const { tenant, isAdmin } = jwt.decode(tokenMatch[1]) as any;
+
+    if (isAdmin) {
+      return next()
```

Comment:
I would like to make a log statement for auditing showing that an admin accessed a particular tenant domain and path.